v21.1/movr-flask-setup.md

```diff
@@ -28,6 +28,7 @@ For debugging and development purposes, you can use the [`cockroach demo`](cockr
     ~~~ shell
     $ cockroach demo \
     --nodes=9 \
+    --insecure \
```

Comment:
Is this truly needed?

Reply:
I don't think we need this flag for the sample app.

Reply:
Below, we should instead update this:

```
    {% include copy-clipboard.html %}
    ~~~ shell
    $ cockroach sql --insecure --url='postgres://demo:<demo_password>@127.0.0.1:26257?sslmode=require' -f dbinit.sql
    ~~~

```

to this

```
    {% include copy-clipboard.html %}
    ~~~ shell
    $ cockroach sql --url='postgres://demo:<demo_password>@127.0.0.1:26257?sslmode=require' -f dbinit.sql
    ~~~
```